src/manager.cc

```diff
@@ -438,11 +438,19 @@ bool fastcat::Manager::ConfigJSDBusFromYaml(YAML::Node node)
     } else if (0 == device_class.compare("Ild1900")) {
       device = std::make_shared<Ild1900>();
 
-    } else if (0 == device_class.compare("Egd_Actuator")) {
-      device = std::make_shared<EgdActuator>();
+    } else if (0 == device_class.compare("Actuator")) {
+      WARNING("Starting in v0.12.0, Platinum device support has been added to Fastcat!");
```

Comment:
I do not think this is going to be super helpful anymore. The bus topologies need to be updated as of right now because the `egd_`* parameters are not present anymore. So why not update `device_class` as well if we are updating other parts of the yaml?